src/quotations/strategies/find-quotation/find-by-status.ts

```diff
@@ -0,0 +1,13 @@
+import { Injectable } from '@nestjs/common';
+import { IFindQuotationStrategy } from './find-quotation-strategy.interface';
+import { Quotation } from '@/quotations/models/quotations.model';
+import { QuotationStatus } from '@/shared/enums/quotation-status.enum';
+
+@Injectable()
+export class FindQuotationByStatus implements IFindQuotationStrategy {
+  find(quotationStatus: QuotationStatus): Promise<Quotation[] | null> {
+    return Quotation.findAll({
+      where: { quotationDate: quotationStatus },
```

Comment:
Check again about the find condition (where), you used the wrong condition "quotationDate" but the strategy is find by status